Tests/EventDispatcherTests.swift

```diff
@@ -0,0 +1,198 @@
+import Quick
+import Nimble
+import Clappr
+
+class EventDispatcherTests: QuickSpec {
+
+    override func spec() {
+        describe("EventDispatcher") {
+
+            var eventDispatcher: EventDispatcher!
+            var callbackWasCalled: Bool!
+
+            let eventName = "some-event"
+            let callback: EventCallback = { _ in
+                callbackWasCalled = true
+            }
+
+            beforeEach {
+                eventDispatcher = EventDispatcher()
+                callbackWasCalled = false
+            }
+
+            describe("on") {
```

Comment:
According to the Project conventions, we should add the # prefix on the Method Name.

``context("#on")``